rollup.config.js

```diff
@@ -1,7 +1,26 @@
-module.exports = {
-    input: 'examples/web-components/platform.js',
-    output: {
-        file: 'examples/web-components/bundle.js',
-        format: 'cjs'
+import typescript from '@rollup/plugin-typescript'
+import { terser } from 'rollup-plugin-terser'
+
+function config(input, filePrefix) {
+    return {
+        input,
+        output: [{
+            file: `lib/${filePrefix}.js`,
+            format: 'es',
+            sourcemap: true
+        },
+        {
+            file: `lib/${filePrefix}.min.js`,
+            format: 'es',
+            sourcemap: false,
+            plugins: [terser()]
+
+        }],
+        plugins: [typescript()]
     }
 };
+
+export default [
+    config('src/index.ts', 'index'),
```

Comment:
I believe for node the entry point should not be index.ts, but node-realm.ts

Reply:
but this is probably problematic because of the current structure, we definitely need the node version since that's what we use from the jest tests.

Reply:
I think the issue here is that the existing usage of the lib assumes 3 files to be available to import from, index, browser-realm and node-realm, in the future, we might want to do something more... but I think at least we should preserve that part.